integration-tests/plugins/__tests__/api-key/admin/api-key.spec.ts

```diff
@@ -39,6 +40,9 @@ describe("API Keys - Admin", () => {
 
   beforeEach(async () => {
     await createAdminUser(dbConnection, adminHeaders)
+
+    // Used for testing cross-module authentication checks
+    regionService.createDefaultCountriesAndCurrencies()
```

Comment:
question: Don't we need an `await` here?

Reply:
Good catch, we do!